src/cli.ts

```diff
@@ -3,12 +3,50 @@
 import chalk from "chalk";
 import program from "commander";
 import fs from "fs-extra";
+import { RewriteRule, RewriteRules } from "./rewrite";
 import { RecordReplayServer } from "./server";
 
 function commaSeparatedList(value: string) {
   return value ? value.split(",") : [];
 }
 
+const RE_REWRITE_RULE = /s\/(.+(?<!\\))\/(.+(?<!\\))\/([gims]*)/;
```

Comment:
**question** Should we have a comment here describing that this is sed-style regex? I know it's mentioned on line 17, but feels more relevant to have with the definition of the rule.

Do we want to link to a documentation of the style somewhere?